src/commonMain/kotlin/com/hedvig/authlib/network/StartLoginRequest.kt

```diff
@@ -34,4 +34,10 @@ internal fun HttpRequestBuilder.buildStartLoginRequest(
 
     contentType(ContentType.Application.Json)
     setBody(body)
+    when (loginMethod) {
+        LoginMethod.SE_BANKID -> {
+            headers["hedvig-bankid-v6"] = "true"
+        }
+        else -> {}
+    }
```

Comment:
I think something like this might just work here
```suggestion
    if (loginMethod is LoginMethod.SE_BANKID) {
        headers["hedvig-bankid-v6"] = "true"
    }
```